src/components/FlowStep/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from "react"
+import { Box, Icon, useColorModeValue } from "@chakra-ui/react"
+import { BodyMd, BodySm, LabelLg, LabelSm, LabelXs } from "../Typography"
+import { IoArrowForwardCircleSharp } from "react-icons/all"
+
+export type FlowStepStatus = "active" | "inactive" | "complete"
+export type FlowStepPosition = "vertical" | "horizontal"
+export type FlowStepSize = "sm" | "lg"
+
+export interface FlowStepProps {
+  status?: FlowStepStatus
+  position?: FlowStepPosition
+  size?: FlowStepSize
+  preTitle: string
+  title: string
+  description?: string | JSX.Element
+  hideDescription?: boolean
+  hideDescriptionArrow?: boolean
+}
+
+export const FlowStep: FC<FlowStepProps> = ({
```

Comment:
I think it would be a good idea to restyle this component in a chakra-way so adding a theme, maybe using multiStyleConfig (similar to ChecklistItem for example)